src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/Event/PrepareMultipleModelsActionEvent.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * PHP version 5
+ *
+ * @package    generalDriver
+ * @author     David Molineus <david.molineus@netzmacht.de>
+ * @copyright  The MetaModels team.
+ * @license    LGPL.
+ * @filesource
+ */
+
+namespace ContaoCommunityAlliance\DcGeneral\Contao\View\Contao2BackendView\Event;
+
+use ContaoCommunityAlliance\DcGeneral\Action;
+use ContaoCommunityAlliance\DcGeneral\EnvironmentInterface;
+use ContaoCommunityAlliance\DcGeneral\Event\AbstractActionAwareEvent;
+
+/**
+ * Class PrepareMultipleModelsActionEvent.
+ *
+ * This event gets emitted when a multiple action shall be handled. A list of affected model ids are given. Use this
+ * event to apply permission based filtering.
+ *
+ * @package DcGeneral\Contao\View\Contao2BackendView\Event
+ */
+class PrepareMultipleModelsActionEvent extends AbstractActionAwareEvent
+{
+    const NAME = 'dc-general.view.contao2backend.prepare-multiple-models-action';
+
+    /**
+     * The model ids.
+     *
+     * @var array
+     */
+    private $modelIds;
+
+    /**
+     * The submit action.
+     *
+     * @var string
+     */
+    private $submitAction;
+
+    /**
+     * Create a new instance.
+     *
+     * @param EnvironmentInterface $environment  The environment.
+     * @param Action               $action       The called action.
+     * @param array                $modelIds     The list of model ids being parsed.
+     * @param string               $submitAction The submit action name.
+     */
+    public function __construct(EnvironmentInterface $environment, Action $action, array $modelIds, $submitAction)
+    {
+        parent::__construct($environment, $action);
+
+        $this->modelIds     = $modelIds;
```

Comment:
Are the ids here instances or strings? The phpDoc should reflect this either as `@param ModelId[]`or as `@param string[]` rather than using the generic `@param array` here.